AEPCore/Sources/eventhub/EventType.swift

```diff
@@ -34,6 +35,7 @@ public class EventType: NSObject {
     public static let identity = "com.adobe.eventType.identity"
     public static let lifecycle = "com.adobe.eventType.lifecycle"
     public static let location = "com.adobe.eventType.location"
+    public static let media = "com.adobe.eventType.media"
```

Comment:
the changes look good to me with one question here, do we need to include this event type since the sources are not in Core?

Reply:
I included it because it's already in Android Core. Android Media does use `EventType.Media` from Core even though the event sources are defined internally to the extension.